### DISCUSSION TITLE
SVG

### DISCUSSION THREAD

**Discussion by Aytharn** [fsprojects/Avalonia.FuncUI#354] [Q&A]
I am always quite a bit all over the place how a UI library is missing a simple, clear, straightforward way to load an SVG icon in 2023.

Being a library based on Skia, which is basically itself an SVG renderer on hormones, it's quite shocking.

**Comment by FoggyFinder** [comment]
You can use https://github.com/wieslawsoltes/Svg.Skia

**Comment by Aytharn** [comment]
already tried: it works in XAML using Avalonia v11, using SVG node, but it doesn't in FuncUI, no documentation, and ChatGPT neither could find a working solution

**Comment by JordanMarr** [comment]
> I am always quite a bit all over the place how a UI library is missing a simple, clear, straightforward way to load an SVG icon in 2023.
> 
> Being a library based on Skia, which is basically itself an SVG renderer on hormones, it's quite shocking.

We accept PRs.

**Comment by JaggerJo** [comment]
https://github.com/wieslawsoltes/Svg.Skia

<img width="1057" alt="Screenshot 2023-10-22 at 13 20 05" src="https://github.com/fsprojects/Avalonia.FuncUI/assets/13090415/3a220d96-1f78-484a-bb0f-63dd36965975">

<img width="685" alt="image" src="https://github.com/fsprojects/Avalonia.FuncUI/assets/13090415/c19cfe88-4daa-4c4e-9f0c-a234a16933cb">

**Comment by Aytharn** [comment]
Thank you!!
